synapse/lib/storm.py

```diff
@@ -2164,6 +2176,10 @@ def _print_optarg(self, names, argdef):
         helpstr = argdef.get('help', 'No help available.')
         defval = argdef.get('default', s_common.novalu)
         if defval is not s_common.novalu and oact not in ('store_true', 'store_false'):
+            if isinstance(defval, (tuple, list, dict)):
+                defval = pprint.pformat(defval, indent=34, width=100)
```

Comment:
Where does 34 come from?

Reply:
The ljust(30) (used for base variable) and an additional 4 for indent.